examples/seismic/tutorials/14_creating_synthetics.ipynb

```diff
@@ -48,7 +48,8 @@
      "name": "stdout",
```

Comment:
<p>you have your local env here printed in the warning</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/devitocodes/devito/pull/1895/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='32'/>